development/compiling/introduction_to_the_buildsystem.rst

```diff
@@ -222,6 +222,18 @@ features to include/disable.
 Check the output of ``scons --help`` for details about each option for
 the version you are willing to compile.
 
+Cleaning generated files
+------------------------
+
+Sometimes, you may encounter an error due to generated files being present. You
+can remove them by using ``scons --clean <options>``, where ``<options>`` is the
+list of build options you've used to build Godot previously.
+
+Alternatively, you can use ``git clean -dfxi`` which will clean build artifacts
```

Comment:
I use `-fxd`, either to remember ("fixed"). Could be `-fixd` with the `i` switch too.